cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -206,9 +208,14 @@ class ReporterCommand : CliktCommand(
             @Suppress("TooGenericExceptionCaught")
             try {
                 val options = reportOptionsMap[reporter.reporterName.toUpperCase()].orEmpty()
-                reporter.generateReport(file.outputStream(), input, options)
 
-                println("Created '${reporter.reporterName}' report:\n\t$file")
+                println("Creating the '${reporter.reporterName}' report at '$file'.")
+
+                val duration = measureTime {
+                    reporter.generateReport(file.outputStream(), input, options)
+                }
+
+                println("Creating the '${reporter.reporterName}' report took ${duration.inSeconds}s.")
```

Comment:
Generating the report can potentially create a log output, therefore I think we should repeat the file location here. Also the wording somehow puts the emphasis on the duration not the report. What do you think of this?
`"Created the '${reporter.reporterName}' report at '$file' (took ${duration.inSeconds}s)."`

Reply:
I decided to move the new output to a summary in the end, so that all durations can be seen at a single glance.